app/api/server/v1/users.js

```diff
@@ -569,3 +569,18 @@ API.v1.addRoute('users.removePersonalAccessToken', { authRequired: true }, {
 		return API.v1.success();
 	},
 });
+
+API.v1.addRoute('users.active', { authRequired: true }, {
```

Comment:
Can we make a slightly more clear word here?  We have the whole active property that determines if an account is enabled.  So I wonder if there will be confusion?

Maybe not?  I know we did change the i18n to now be activated instead of active.

Maybe users.presence?  Idk just throwing out there, if clear enough ignore me.  :)

Reply:
I agree. I was never happy with this name :joy: `users.presence` looks better for me.